servlet/jakarta/build.gradle

```diff
@@ -0,0 +1,114 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
+}
+
+description = "gRPC: Jakarta Servlet"
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+// Set up classpaths and source directories for different servlet tests
+configurations {
+    itImplementation.extendsFrom(implementation)
+    jettyTestImplementation.extendsFrom(itImplementation)
+    tomcatTestImplementation.extendsFrom(itImplementation)
+    undertowTestImplementation.extendsFrom(itImplementation)
+}
+
+sourceSets {
+    undertowTest {
+        java {
+            include '**/Undertow*.java'
+        }
+    }
+    tomcatTest {
+        java {
+            include '**/Tomcat*.java'
+        }
+    }
+    // Only run these tests if java 11+ is being used
+    if (JavaVersion.current().isJava11Compatible()) {
+        jettyTest {
+            java {
+                include '**/Jetty*.java'
+            }
+        }
+    }
+}
+
+// Mechanically transform sources from grpc-servlet to use the corrected packages
+def migrate(String name, String inputDir, SourceSet sourceSet) {
```

Comment:
Can you also migrate `io/grpc/servlet/*.java` to  something like `io/grpc/servletjakarta/*.java`? Different APIs with exactly the same class and package name will potentially cause trouble.

Reply:
Yes I'll get that resolved this week, sorry for the delay.

Reply:
Thanks. BTW, there are many possibilities for the package naming like, but not limited to, the following. As a user, just take whatever is most convenient for you.

| Java EE 8      | Jakarta EE |
| ----------- | ----------- |
| io.grpc.servlet |  io.grpc.servletjakarta |
| io.grpc.servlet |  io.grpc.servlet.jakarta |
| io.grpc.servlet.javaee8   | io.grpc.servlet |
| io.grpc.servlet.javaee8 | io.grpc.servlet.jakarta |
| io.grpc.servlet.javax | io.grpc.servlet.jakarta |

Reply:
I went with `io.grpc.servlet` -> `io.grpc.servlet.jakarta`.